src/random.cr

```diff
@@ -289,26 +289,39 @@ module Random
     end
   end
 
-  # Returns a StaticArray filled with random integers.
-  # Note that this only works with StaticArray of integer types.
-  #
-  # ```
-  # Random.new.rand(StaticArray(Int32, 4))  # => StaticArray[4795590, -1987398873, 1859211601, 546845569]
-  # Random.new.rand(StaticArray(UInt16, 2)) # => StaticArray[49247, 53674]
-  # ```
-  def rand(type : StaticArray(T, N).class) forall T, N
-    {%
-      if T != Int8 && T != UInt8 &&
-         T != Int16 && T != UInt16 &&
-         T != Int32 && T != UInt32 &&
-         T != Int64 && T != UInt64
-        raise "Random::Secure#random_static argument must be StaticArray of integer type"
-      end
-    %}
+  {% for type, values in {
+                           "Int8".id   => %w(20 -66 89 19),
+                           "UInt8".id  => %w(186 221 127 245),
+                           "Int16".id  => %w(-32554 32169 -20152 -7686),
+                           "UInt16".id => %w(39546 44091 2874 17348),
+                           "Int32".id  => %w(1870830079 -1043532158 -867180637 -1216773590),
+                           "UInt32".id => %w(3147957137 4245108745 2207809043 3184391838),
+                           "Int64".id  => %w(4438449217673515190 8514493061600538358 -4874671083204037318 -7825896160729246667),
+                           "UInt64".id => %w(15004487597684511003 12027825265648206103 11303949506191212698 6228566501671148658),
+                         } %}
```

Comment:
Since these numbers will change anyway isn't it pointless to include them in docs?

Reply:
I wanted to include example values. How would you do it otherwise?

Reply:
I'd write the values like `<random {{type}} integer>`.

Reply:
I think a real example is better